app/Http/Requests/User/SetEmailRequest.php

```diff
@@ -15,7 +15,7 @@ class SetEmailRequest extends BaseApiRequest
 
 	public function authorize(): bool
 	{
-		return Gate::check(UserPolicy::CAN_EDIT_SETTINGS, User::class);
+		return Gate::check(UserPolicy::MAY_EDIT_OWN_SETTINGS, User::class);
```

Comment:
See some comments above. We need a right `UserPolicy::CAN_EDIT` here. At the moment, I would go with a single user-edit right. We can still add more fine granular rights like `UserPolicy::CAN_EDIT_EMAIL` later.